docs/docs/types_and_serializers.rst

```diff
@@ -0,0 +1,121 @@
+=====================================
+Tags and Descriptions
+=====================================
+
+Unitxt employ different tools for serializing data into textual
+format. One of this mechanism is the Type-Serializers mechanism which assign serialization to specific types.
+For example consider the following ``typing types``:
+
+.. code-block:: python
+
+    Text = NewType("Text", str)
```

Comment:
What do I need to import for NewType?

Reply:
It's `typing` class. Will add its import.